lib/3rd_party_adapters/lv_conf.h

```diff
@@ -87,8 +87,8 @@
  *It removes the need to manually update the tick with `lv_tick_inc()`)*/
 #define LV_TICK_CUSTOM 1
 #if LV_TICK_CUSTOM
-    #define LV_TICK_CUSTOM_INCLUDE "Arduino.h"         /*Header for the system time function*/
-    #define LV_TICK_CUSTOM_SYS_TIME_EXPR (millis())    /*Expression evaluating to current system time in ms*/
+    #define LV_TICK_CUSTOM_INCLUDE <chrono> /*Header for the system time function*/
+    #define LV_TICK_CUSTOM_SYS_TIME_EXPR (std::chrono::round<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count()) /*Expression evaluating to current system time in ms*/
```

Comment:
🛑 This is not thread safe. ⚠️ See [this warning](https://github.com/espressif/esp-idf/blob/bbd8d13f930705051afa883691bc8389eefc48a1/components/newlib/locks.c#L135).